root/documentation/compara.conf

```diff
@@ -254,9 +254,25 @@
       </aligned>
```

Comment:
Add a note that aligned=1 overrides sequence=none for aligned as well as for sequence

Reply:
Thank Brandon. I have made the suggested changes